modules/src/core/ics04_channel/events.rs

```diff
@@ -8,21 +8,16 @@ use crate::core::ics02_client::height::Height;
 use crate::core::ics04_channel::error::Error;
 use crate::core::ics04_channel::packet::Packet;
 use crate::core::ics24_host::identifier::{ChannelId, ConnectionId, PortId};
-use crate::events::{
-    extract_attribute, maybe_extract_attribute, Error as EventError, IbcEvent, IbcEventType,
-    RawObject,
-};
+use crate::events::{Error as EventError, IbcEvent, IbcEventType};
 use crate::prelude::*;
 
-use super::packet::parse_timeout_height;
-
 /// Channel event attribute keys
-const HEIGHT_ATTRIBUTE_KEY: &str = "height";
-const CONNECTION_ID_ATTRIBUTE_KEY: &str = "connection_id";
-const CHANNEL_ID_ATTRIBUTE_KEY: &str = "channel_id";
-const PORT_ID_ATTRIBUTE_KEY: &str = "port_id";
-const COUNTERPARTY_CHANNEL_ID_ATTRIBUTE_KEY: &str = "counterparty_channel_id";
-const COUNTERPARTY_PORT_ID_ATTRIBUTE_KEY: &str = "counterparty_port_id";
+pub const HEIGHT_ATTRIBUTE_KEY: &str = "height";
+pub const CONNECTION_ID_ATTRIBUTE_KEY: &str = "connection_id";
+pub const CHANNEL_ID_ATTRIBUTE_KEY: &str = "channel_id";
+pub const PORT_ID_ATTRIBUTE_KEY: &str = "port_id";
+pub const COUNTERPARTY_CHANNEL_ID_ATTRIBUTE_KEY: &str = "counterparty_channel_id";
+pub const COUNTERPARTY_PORT_ID_ATTRIBUTE_KEY: &str = "counterparty_port_id";
```

Comment:
These are Tendermint-specific, so we might move them out of `ibc` crate later on.

Reply:
Right, I was also thinking about these and I didn't remove them for now because basecoin  needs (and arguably other chains using the modules would need) conversions from `IbcEvent` to `tendermint_abci::Event` (which use these `const`s). We need to figure out a good place for these conversions.